doc/classes/DisplayServer.xml

```diff
@@ -1088,18 +1088,18 @@
 			<return type="Image" />
 			<param index="0" name="screen" type="int" default="-1" />
 			<description>
-				Returns screenshot of the [param screen].
+				Returns a screenshot of the [param screen].
 				[b]Note:[/b] This method is implemented on Linux (X11), macOS, and Windows.
-				[b]Note:[/b] On macOS, this method requires "Screen Recording" permission, if permission is not granted it will return desktop wallpaper color.
+				[b]Note:[/b] On macOS, this method requires the "Screen Recording" permission. If permission is not granted, this method returns an [Image] filled with the desktop wallpaper's color.
```

Comment:
> desktop wallpaper color.

Actually it's an image app own windows on top of desktop wallpaper (no icons/widgets, or other app windows aren't visible), not filled with color.

Reply:
Sorry, missed this, here's an example of what app capture vs. real screen contents:

Real screen:

<img width="1504" alt="Screenshot 2025-04-04 at 00 11 35" src="https://github.com/user-attachments/assets/6630fa85-322d-4c56-b1be-746cea9343b5" />

What's app without permission capture:
<img width="1504" alt="a" src="https://github.com/user-attachments/assets/5435ad4c-0843-4cd3-8d37-bbc8254df1a3" />

Reply:
Alright, thank you. I have updated the note to be hopefully more accurate.